liquibase/scripts/fix_liquibase_snapshots.sh

```diff
@@ -17,6 +17,6 @@
 #  The updated snapshots are written to the openmrs-core/liquibase/snapshots folder.
 #
 
-openmrs_version=2.4.0-SNAPSHOT
+openmrs_version=2.6.0-SNAPSHOT
```

Comment:
we can extract the current OpenMRS version directly from the `pom.xml` file, So we no need to manually update the OpenMRS version during the release of a new version.